WordPress/wporg-theme-directory
# Tools: Add RTLCSS step to theme build process

This PR adds RTLCSS to the webpack build process (which requires a custom webpack.config.js, but we can inherit most of the configuration). This also fixes an issue with ratings on RTL sites.

Fixes #47, See https://github.com/WordPress/wporg-main-2022/issues/124

### Screenshots



| Before | After |
|--------|-------|
| <img width="373" alt="Screenshot 2024-05-01 at 4 21 29 PM" src="https://github.com/WordPress/wporg-theme-directory/assets/541093/d982cbde-54d5-4914-bc9a-c0753a09c3e5">  | <img width="400" alt="Screenshot 2024-05-09 at 2 55 55 PM" src="https://github.com/WordPress/wporg-theme-directory/assets/541093/6a3622d3-f53c-43d3-b7a3-e86cb4d585f0"> |

### How to test the changes in this Pull Request:

1. Update the packages with `yarn`
2. Run `yarn build:theme`
3. You should see `-rtl.css` files in the build output & build folder
4. View a single theme on an RTL site (using a sandbox)
5. The rating bar and stars should be appropriately RTL too
